packages/lodestar/test/e2e/chain/bls/multithread.test.ts

```diff
@@ -88,7 +88,7 @@ describe("chain / bls / multithread queue", function () {
       isValidPromiseArr.push(pool.verifySignatureSets(sets, {batchable: true}));
     }
 
-    await expect(isInvalidPromise).to.rejectedWith("BLST_ERROR");
+    await expect(isInvalidPromise).to.rejectedWith("BLST_INVALID_SIZE");
```

Comment:
The test was failing because the error is now `BLST_INVALID_SIZE`. 

The test is supposed to assert that _If the first signature is invalid it should not make the rest throw_ which is still true, but I am not sure if the change in the error is a sign of something else being wrong. Any thoughts?

Reply:
Not sure how the test was passing before, I believe the `signature` on L84 should be 96 bytes...
So there is an "invalid size" present.

Reply:
Ok. I take that to mean that the upgrade somewhat made the error more specific. Which is not a bad thing...